pyproject.toml

```diff
@@ -71,7 +71,8 @@ tutorials = [
     "jupyter",
     "seaborn",
     "matplotlib",
-    "datasets>=2.8.0,<2.11.0",
+    "datasets>=2.8.0",
+    "scikit-learn>=1.0.1",
```

Comment:
came back here when the tests failed: shoudn't this be in the package dependency now? Probably we want it here

Reply:
you're right! Imo, sklearn is an alright package dependency - I vote we move it to the base dependencies.

Reply:
Let's keep it out. It is an integration not a core feature.